packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -153,7 +153,12 @@ export class WebsocketConnector {
                 } else if (action === 'connectivityProbe') {
                     // no-op
                 } else {
-                    this.attachHandshaker(connection)
+                    if (this.localPeerDescriptor !== undefined) {
```

Comment:
Worth adding a comment why the `localPeerDescription` can be `undefined` (e.g. similar info which was written here: https://github.com/streamr-dev/network/pull/2328#issuecomment-1908062807)